packages/api/src/auth/github/github.service.spec.ts

```diff
@@ -37,28 +37,28 @@ describe('Github controller', () => {
     it('Returns already existing user when `githubId` matches some record in db', async () => {
       const dbUser = createUser(111);
 
-      usersRepository.getByGithubId = jest.fn().mockResolvedValue(dbUser);
+      usersService.getByGithubId = jest.fn().mockResolvedValue(dbUser);
 
       const githubUser = createObjectMock<UserFromGithub>({ githubId: dbUser.githubId });
 
       const result = await service.authorizeUser(githubUser);
 
-      expect(usersRepository.create).not.toHaveBeenCalled();
+      expect(usersService.create).not.toHaveBeenCalled();
```

Comment:
This is unnecessary. It's testing implementation details. It's not important in tests HOW something happened but WHAT. So `expect(result).toEqual(dbUser);` is enough.